docs/docs/configuration/queryable-projections.mdx

```diff
@@ -76,3 +74,48 @@ public static partial class CarMapper
     private static partial CarDto Map(Car car);
 }
 ```
+
+## User-implemented mapping methods
+
+Mapperly tries to inline user-implemented mapping methods.
+For this to work, user-implemented mapping methods need satisfy certain limitations:
+
+- Only expression-bodied methods can be inlined.
+- The body needs to follow the expression tree [limitations](https://learn.microsoft.com/en-us/dotnet/csharp/advanced-topics/expression-trees/#limitations).
```

Comment:
Nitpick: Would add the link over the whole "expression tree limitations" text

Reply:
Improved